torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -202,6 +217,14 @@ The explanation needs to be clear on why this is needed. Here are some good exam
 
     const filteredLabels = labelsToAdd.filter((l: string) => repoLabels.has(l));
     const invalidLabels = labelsToAdd.filter((l: string) => !repoLabels.has(l));
+    const ciflowLabels = labelsToAdd.filter((l: string) => l.startsWith("ciflow/"));
```

Comment:
Instead of blocking "ciflow/" specifically, why not explicitly whitelist the labels that we thing people without write permission should be able to use?